packages/playwright-core/src/server/injected/injectedScript.ts

```diff
@@ -132,7 +135,12 @@ export class InjectedScript {
     return globalThis.eval(expression);
   }
 
-  parseSelector(selector: string): ParsedSelector {
+  testIdAttributeNameForStrictErrorAndConsoleCodegen(): string {
+    return this._testIdAttributeNameForErrorAndConsoleCodegen;
+  }
+
+  parseSelectorBestEffortForTools(selector: string): ParsedSelector {
+    // Don't use this for strict parsing, only server can parse reliably.
```

Comment:
Can we explain here why? Otherwise it is unclear, because the method parses everything perfectly fine.